README.md

```diff
@@ -19,28 +20,41 @@ A really simple, lightweight Angular pipe for converting a date string into a ti
 |546 days+                  | X years ago            |
 
 ## Installation
-```npm install time-ago-pipe --save```
+
+```
+npm install time-ago-pipe --save
+```
```

Comment:
Why change the formatting of the one-liner?

Reply:
Too long ago - can't remember. Probably it was the [markdown linter](https://github.com/markdownlint/markdownlint)